### DISCUSSION TITLE
Any reason for our RAG system to give out such response in chat

### DISCUSSION THREAD

**Discussion by binxushy** [infiniflow/ragflow#4092] [Q&A]
Version: v0.14.0

![image](https://github.com/user-attachments/assets/78d9c648-0712-49b9-aba7-fefc0f9758a7)

**Comment by KevinHuSh** [comment]
What's your RAGFlow version?

**Reply by binxushy** [reply]
v 0.14.0

**Reply by KevinHuSh** [reply]
It's fixed by v0.15.0.
